webservices/resources/legal.py

```diff
@@ -121,27 +98,30 @@ def get(self, q='', from_hit=0, hits_returned=20, **kwargs):
             "admin_fines": case_query_builder,
         }
 
-        if kwargs.get('type', 'all') == 'all':
+        if kwargs.get("type", "all") == "all":
             doc_types = ALL_DOCUMENT_TYPES
         else:
-            doc_types = [kwargs.get('type')]
+            doc_types = [kwargs.get("type")]
 
             # if doc_types is not in one of ALL_DOCUMENT_TYPES
             # then reset type = all (= ALL_DOCUMENT_TYPES)
             if doc_types[0] not in ALL_DOCUMENT_TYPES:
                 doc_types = ALL_DOCUMENT_TYPES
 
         hits_returned = min([200, hits_returned])
+        print(hits_returned)
```

Comment:
I think this `print` statement needs to be removed

Reply:
Thanks for flagging. Will remove this line.